app/components/worship-service-select.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { restartableTask, timeout } from 'ember-concurrency';
+
+export default class WorshipServiceSelectComponent extends Component {
+  @service fastboot;
+  @service store;
+  worshipServices;
+
+  @restartableTask
+  *loadWorshipServicesTask(searchParams = '') {
+    yield timeout(500);
+
+    const query = {
+      sort: 'name',
+    };
+
+    if (searchParams.trim().length !== '') {
```

Comment:
the `.length` part isn't needed anymore. This will always be true at the moment I think.